### PR TITLE
fix(jest): Includes @jest/globals

### PR DESCRIPTION
**Pull request by grosch**
mock.ts was calling jest methods but it didn't import the jest object.

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/ngneat/spectator/blob/master/CONTRIBUTING.md#commit
- [x] Tests for the changes have been added (for bug fixes / features)
- [X] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[x] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?
The mocks.ts file wasn't including @jest/globals, which is now required.

Issue Number: 560

## What is the new behavior?
Just imports so that calling tests don't fail.

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

### CODE REVIEW

File: projects/spectator/jest/src/lib/mock.ts [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 import { FactoryProvider, AbstractType, Type } from '@angular/core';
 import { installProtoMethods, CompatibleSpy, SpyObject as BaseSpyObject } from '@ngneat/spectator';
+import { jest } from '@jest/globals'
```
Comment (NetanelBasal) [inline projects/spectator/jest/src/lib/mock.ts:5]: Is this a breaking change? It's from a specific version only or it'll throw in older jest versions?

Comment (grosch) [reply projects/spectator/jest/src/lib/mock.ts:5]: According to the [CHANGELOG.md](https://github.com/facebook/jest/blob/main/CHANGELOG.md#2550) from jest, it looks like that file appeared in version 25.5.0

Comment (NetanelBasal) [reply projects/spectator/jest/src/lib/mock.ts:5]: so it should be a peer dependency with this version and above. Please remove the release changes and the git tag. I will do it.

Comment (grosch) [reply projects/spectator/jest/src/lib/mock.ts:5]: Thanks!  I think I did what you were asking but not 100% sure.  I'm kinda new to giving PR's like this :)

**Comment by grosch** [comment]
@NetanelBasal Any updates?

File: projects/spectator/package.json [hunk 2]
```diff
@@ -2,7 +2,7 @@
   "name": "@ngneat/spectator",
```
Comment (NetanelBasal) [inline projects/spectator/package.json:2]: Please revert this and add the right peer dependency for jest.

Comment (grosch) [reply projects/spectator/package.json:2]: I added the peer dependency. Not sure what you mean by revert that though.  I didn't change the name.

diff --git a/projects/spectator/package.json b/projects/spectator/package.json
index 5c2c9d2..1cfe23e 100644
--- a/projects/spectator/package.json
+++ b/projects/spectator/package.json
@@ -2,7 +2,7 @@
   "name": "@ngneat/spectator",
   "description": "A powerful tool to simplify your Angular tests",
   "author": "Netanel Basal <netanel7799s@gmail.com>",
-  "version": "11.0.1",
+  "version": "11.0.2",
   "license": "MIT",
   "repository": {
     "type": "git",
@@ -31,7 +31,8 @@
   "peerDependencies": {
     "@angular/common": ">= 14.0.0",
     "@angular/router": ">= 14.0.0",
-    "@angular/animations": ">= 14.0.0"
+    "@angular/animations": ">= 14.0.0",
+    "jest": ">= 25.5.0"
   },
   "schematics": "./schematics/collection.json"
 }

Comment (NetanelBasal) [reply projects/spectator/package.json:2]: I mean revert `  "version": "11.0.2",`

Comment (grosch) [reply projects/spectator/package.json]: Done

**Comment by NetanelBasal** [comment]
Something is bothering me here. We're [using](https://github.com/ngneat/spectator/blob/master/package.json#L73)  `"jest": "28.0.0"`, so how does it work if you say this change is from v25.5.0?

**Comment by grosch** [comment]
I think it’s the difference between whether or not you’re using the ESM stuff.  You asked me when the @jest/globals file appeared, and that was in 25.5.0

Have you tried the sample I sent?  You should immediately see the failure yourself, and that just including that file fixes thing, and doesn’t break anything else, from what I can tell.

**Comment by NetanelBasal** [comment]
I'll try your example later today.

**Pull request closed without merging**